pages/models.py

```diff
@@ -340,6 +350,11 @@ def get_followers(element):
         """
         return get_user_model().objects.filter(follows__element=element)
 
+    @staticmethod
+    def followed_elements(user):
+        page_element_slugs = user.follows.all().values_list('element__slug', flat=True)
+        return PageElement.objects.filter(slug__in=page_element_slugs)
```

Comment:
That seems odd that the `FollowManager` returns a queryset of `PageElement`. It might be best to move this method in `PageElementManager` and call it `followed_by(user)`.